info.go

```diff
@@ -391,6 +397,29 @@ func (pi *ProgramInfo) Instructions() (asm.Instructions, error) {
 	return insns, nil
 }
 
+// JitedSize returns the size of the program's JIT-compiled machine code in bytes, which is the
+// actual code executed on the host's CPU. This field requires the BPF JIT compiler to be enable.
+//
+// Available from 4.13. Reading this metadata requires CAP_SYS_ADMIN or equivalent.
+//
+// The bool return value indicates whether the program has been JIT-compiled, or whether this
+// field is available.
+func (pi *ProgramInfo) JitedSize() (uint32, bool) {
+	return pi.jitedSize, pi.jitedSize > 0
+}
+
+// TranslatedSize returns the size of the program's translated instructions in bytes, after it has
+// been verified and rewritten by the kernel.
+//
+// Available from 4.13. Reading this metadata requires CAP_SYS_ADMIN or equivalent.
```

Comment:
If I understand the code comments earlier in the file correctly, `pi.insns` should be populated if the process has CAP_BPF. CAP_SYS_ADMIN is only required for fetching the program's BTF blob, which isn't needed in TranslatedSize().

Reply:
Ah, you're right. I was originally basing this off of the commit https://github.com/torvalds/linux/commit/1e270976908686ec25fb91b8a34145be54137976#diff-9fb056de263cc4a10925f1474cc0371518419d22bdb4b26d8cfcc7043bfa7883R1292, but it looks like on newer kernels, either `CAP_BPF` or `CAP_SYS_ADMIN` are permitted https://elixir.bootlin.com/linux/v6.1/source/kernel/bpf/syscall.c#L3954.

Reply:
Awesome, thanks for checking!